VideoLocker/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -404,4 +412,11 @@ public void setLastCourseStructureFetch(String courseId, long timestamp) {
         public static final String BACKEND_FACEBOOK = "facebook";
         public static final String BACKEND_GOOGLE = "google-oauth2";
     }
+
+    public static void nukeSharedPreferences() {
```

Comment:
Docstring would be helpful to clarify what we mean by nuke. 

Looks like nuke in our case is just emptying out the user data instead of deleting the shared_prefs folder.